docker-for-mac/multi-arch.md

```diff
@@ -54,3 +54,134 @@ Notice that this time, the `uname -a` output shows `armv7l` and
+
+### Install
+
+1. Download Docker Desktop 2.0.4.0 or higher.
+
+    - [Docker Desktop 2.0.4.0 for Mac](https://docs.docker.com/docker-for-mac/edge-release-notes/)
+    - [Docker Desktop 2.0.4.0 for Windows](https://docs.docker.com/docker-for-windows/edge-release-notes/)
+
+1. Follow the on-screen instructions to complete the installation process. After you have successfully installed Docker Desktop, you will see the Docker icon in your task tray.
+
+1. Click **About Docker Desktop** from the Docker menu and ensure you have installed Docker Desktop version 2.0.4.0 (33772) or higher.
+
+![about-docker-desktop-buildx](./images/desktop-buildx-version.png)
+
+### Build and run multi-architecture images
+
+Run the command `docker buildx ls` to list the existing builders. This displays the default builder, which is our old builder.
+
+```bash
+$ docker buildx ls
+
+NAME/NODE DRIVER/ENDPOINT STATUS  PLATFORMS
+default * docker
+  default default         running linux/amd64, linux/arm64, linux/arm/v7, linux/arm/v6
+```
+
+Create a new builder which gives access to the new multi-architecture features.
+
+```bash
+$ docker buildx create --name mybuilder
+
+mybuilder
+```
+
+Switch to the new builder and inspect it.
+
+```bash
+$ docker buildx use mybuilder
+
+$ docker buildx inspect --bootstrap
+
+[+] Building 2.5s (1/1) FINISHED
+ => [internal] booting buildkit                                                   2.5s
+ => => pulling image moby/buildkit:master                                         1.3s
+ => => creating container buildx_buildkit_mybuilder0                              1.2s
+Name:   mybuilder
+Driver: docker-container
+
+Nodes:
+Name:      mybuilder0
+Endpoint:  unix:///var/run/docker.sock
+Status:    running
+
+Platforms: linux/amd64, linux/arm64, linux/arm/v7, linux/arm/v6
+```
+
+Test the workflow to ensure you can build, push, and run multi-architecture images. Create a simple example Dockerfile, build a couple of image variants, and push them to Docker Hub.
+
+```bash
+$ mkdir test && cd test && cat <<EOF > Dockerfile
+
+FROM ubuntu
+RUN apt-get update && apt-get install -y curl
+WORKDIR /src
+COPY . .
+EOF
+```
+
+```bash
+$ docker buildx build --platform linux/amd64,linux/arm64,linux/arm/v7 -t username/demo:latest --push .
+
+[+] Building 6.9s (19/19) FINISHED
+...
+ => => pushing layers                                                             2.7s
+ => => pushing manifest for docker.io/username/demo:latest                       2.2
+ ```
+
+Where, `username` is a valid Docker username.
+
+>   **Notes:**
+>
+>  - The `--platform` flag informs buildx to generate Linux images for Intel 64-bit, Arm 32-bit, and Arm 64-bit architectures.
```

Comment:
the order for Arm 32-bit and Arm 64-bit is not consistent with the order specified by `--platform`: arm64 then arm/v7. In practice the order doesn't matter though so this is only a stylistic comment.

Reply:
Fixed in the latest commit.